src/main.sh

```diff
@@ -898,7 +898,7 @@ while true; do
 		--lint)                  LINTPKGBUILD=1 ;;
 		--mpr-check|--dur-check) mpr_check; exit $E_OK ;;
 		--print-control)         BUILDPKG=0 PRINTCONTROL=1 IGNOREARCH=1 ;;
-		--print-srcinfo)         BUILDPKG=0 PRINTSRCINFO=1 IGNOREARCH=1 ;;
+		--print-srcinfo|--printsrcinfo)         BUILDPKG=0 PRINTSRCINFO=1 IGNOREARCH=1 ;;
```

Comment:
Can we make this a separate line and show a warning that the user should use `--print-srcinfo` instead? I'm thinking this:

```sh
--printsrcinfo) warning "'--printsrcinfo' will be removed in a future release. Please use '--print-srcinfo' instead.'"; BUILDPKG=0 PRINTSRCINFO1 IGNOREARCH=1 ;;
```